python/mxnet/contrib/onnx/_import/op_translations.py

```diff
@@ -114,7 +114,7 @@ def maximum(attrs, inputs, proto_obj):
         for op_input in inputs[2:]:
             mxnet_op = symbol.maximum(mxnet_op, op_input)
     else:
-        mxnet_op = inputs[0]
+        mxnet_op = symbol.maximum(inputs[0], inputs[0])
```

Comment:
maximum of same element?

Reply:
yes, onnx has a case where if there is only one input, it returns that input itself as output. MXNet needs 2 inputs always